### DISCUSSION TITLE
Is there a way to redirect toRoute() in front w/ inertia?

### DISCUSSION THREAD

**Discussion by benjibenoit** [adonisjs/core#4743] [Help] [answered]
Hey guys,

I'm looking for a way to redirect from react to an adonisjs route using the ".as()". Is this possible?

Something like:
```ts
// On react
export default function Home() {
  function handleClick() {
    router.redirect().toRoute('/home')
  }

  return (
    <div>
      <h1 onCLick={handleClick}>Home</h1>
    </div>
  )
}

// On adonisJS
router.get('/home', [HomeController, 'index']).as('home')
```

Thanks!

**Comment by thetutlage** [comment]
Hey, its covered here. https://docs.adonisjs.com/guides/views-and-templates/inertia#redirects

**Reply by benjibenoit** [reply]
It's not exactly what I'm looking for. I'm looking for something more like https://www.npmjs.com/package/ziggy-js but for AdonisJS

**Comment by Julien-R44** [comment, marked as answer]
We have a package that will likely be released in 1 to 2 weeks that will allow this. But in the meantime, you can use this community package, which seems perfect https://github.com/izzyjs/route#readme 🙂

**Reply by benjibenoit** [reply]
Thanks!
